main.go

```diff
@@ -41,6 +41,7 @@ const (
 
 	dryRunFlagPre118  = "--dry-run=true"
 	dryRunFlagDefault = "--dry-run=client"
```

Comment:
if both the server-side apply and dry-run options are enabled, should we add logic to ensure that the dry-run executes server-side, too? i.e., `--server-dry-run` or `--dry-run=server`?